contrib_bots/lib/github_detail.md

```diff
@@ -0,0 +1,27 @@
+# GitHub detail bot
+This bot links and details issues and pull requests.
+To use it username mention the bot then type an id:
+Ids can be specified in three different forms:
+- Id only: `#2000`
+- Repo and id: `zulip#2000`
+- Owner and repo and id `zulip/zulip#2000`
```

Comment:
Same as line 6. 

Also, you can use a comma instead of the first `and`. A [serial comma] for the second `and` would look nice.